com.unity.render-pipelines.universal/Editor/Camera/URPCameraUI.Skin.cs

```diff
@@ -0,0 +1,91 @@
+using System;
+using System.Linq;
+using UnityEngine;
+using UnityEngine.Rendering.Universal;
+
+namespace UnityEditor.Rendering.Universal
+{
+    internal static partial class UniversalRenderPipelineCameraUI
```

Comment:
i think thats the content for the other file, and you delete this one :)

Reply:
thank you!!